report.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'board'
+require_relative 'burndown'
+require_relative 'helpers'
+
+module Spregen
+  class Report
+
+    include Spregen::Helpers
+
+    attr_accessor :report
+
+    def initialize(configuration = {})
+      Spregen::Config.configure_trello_client
+      @board = Spregen::Board.new(configuration[:board_name])
```

Comment:
I think it's better to create the new instance passing a generic configuration hash instead of board_name. Let the Board class threat how to manage the parameters. That way, we don't need to change how the board is created every time a parameter is added or changed.